### ISSUE TITLE
Request Publishing Ability API

### ISSUE THREAD

**Issue by bryan-robitaille** [cds-snc/platform-forms-client#1226]

**Comment by thiessenp-cds** [comment]
Note: it looks like there is an additional field "goals?" in the Figma design. Easy to add later also if needed.
https://www.figma.com/file/dbezuEglWNQ8qyO64eJJiA/v1.-Form-builder?node-id=958%3A11062&t=0TBnBrDhaWrCG9ug-0

**Comment by craigzour** [comment]
> Note: it looks like there is an additional field "goals?" in the Figma design. Easy to add later also if needed. https://www.figma.com/file/dbezuEglWNQ8qyO64eJJiA/v1.-Form-builder?node-id=958%3A11062&t=0TBnBrDhaWrCG9ug-0

Good catch! I added it in the code https://github.com/cds-snc/platform-forms-client/pull/1242
